domains/priyatosh.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Personal portfolio and Dart/Flutter package publisher.",
+  "owner": {
+    "username": "priyatosh-work",
+    "email": "priyatosh.work@google.com"
+  },
+  "record": {
+    "CNAME": "priyatosh-work.github.io",
+    "TXT": "google-site-verification=wiiXNtcjs3TbnKKNPKGATcZicQ2ldvpy4cZTfa9ZKw0"
+  }
```

you cant combine CNAME with other records unless it was proxied
```suggestion
  },
  "proxied": true
```